docs/quickstart/integrations/aws-lambda/quickstart-guide.mdx

```diff
@@ -348,3 +348,62 @@ handler = Mangum(app)
 
 </BackendTabs.TabItem>
 </BackendTabs>
+
+#### 3.5 Filter additional plugins or extensions {{optional}}
+
+If you are using AWS Lambda plugins, extensions, or anything that adds events to the lambda function (e.g. `serverless-plugin-warmup`), then you may need to filter them and skip passing them to SuperTokens.
```

Comment:
"then you may need to prevent calling SuperTokens with them". Avoid repetition of them and remove redundant sentence.